app/_includes/snippets/install_os.md

```diff
@@ -11,7 +11,7 @@ Finally, you can follow the [Quickstart](#quickstart) to take it from here and c
 Run the following script to automatically detect the operating system and download Kuma:
 
 <div class="language-sh">
-<pre class="no-line-numbers"><code>curl -L https://kuma.io/installer.sh | VERSION={{ page.latest_version }} sh -</code></pre>
+<pre class="no-line-numbers"><code>curl -L https://kuma.io/installer.sh | VERSION={{ include.installer_version }} sh -</code></pre>
```

Comment:
I don't think these are needed anymore. You can use page.latestVersion everywhere this is pulled from: 

https://github.com/kumahq/kuma-website/blob/990729fda76e3258c015d6461b712428afe72361/app/_data/versions.yml#L30

If you update this to `preview` it should work

Reply:
This would be incorrect for any use of `page.latest_version` that isn’t related to the installer script and there are quite a few of those:

```
- **kuma-cp**: at `docker.io/kumahq/kuma-cp:{{ page.latest_version }}`
```

```
You will then find the `kumactl` executable in the `kuma-{{ page.latest_version }}/bin` folder.
```

```
- <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-centos-amd64.tar.gz">CentOS</a>
```

So while it’s definitely still incorrect that these refer to 2.0.0, I think it’s better than referring to `preview` which will definitely fail for all of these cases.

Reply:
Having preview here for the sake of simplicity is better imo. What you are doing is also making single source docs complicated.
We can always push the docker tag on preview builds it's not that complicated

Reply:
I’ve changed this again and use the preview version set in versions.yml now.